### DISCUSSION TITLE
Change camera angle to birds eye view

### DISCUSSION THREAD

**Discussion by kowshik1234** [pokepetter/ursina#608] [Q&A] [answered]
I am having a hard time changing the camera angles and rendering the view I needed. 

The app starts in the following camera view

<img width="536" alt="image" src="https://github.com/pokepetter/ursina/assets/26191169/10e97472-a60d-4f92-8771-b4289d6fcd3e">

However I need it to be as below, 

<img width="554" alt="image" src="https://github.com/pokepetter/ursina/assets/26191169/81383b8f-0772-4856-8a61-2aa91b9b2777">

At the moment, I am using `EditorCamera` and adjusting the view EVERYTIME upon the launch. The solution I need is for the view to adjust automatically upon launch. 

I tried twiddling around the `camera.position = Vec3(x,y,z)` API but it never resulted in the view I wanted. Is there an easy way to find the co-ordinates of the view which is set by EditorCamera and then hard-code them at the start of the app?

**Comment by pokepetter** [comment, marked as answer]
What the EditorCamera does, is to make an Entity as a pivot point, parent the camera to it, and move the camera local z position back. You can then rotate the pivot point and the camera will rotate around that point.

To get the current rotation of the editor camera, just do print(editor_camera.rotation)
You could set the editor camera's rotation to the on creation to avoid adjusting it manually each time.
Example:
```py
from ursina import *
app = Ursina()

Entity(model='plane', scale=8)
EditorCamera(rotation=(45,0,0))
app.run()
```

**Comment by pokepetter** [comment]
Here's how you'd do the same with just an Entity:
```py
from ursina import *
app = Ursina()

Entity(model='plane', scale=8)

pivot = Entity(rotation=(45,0,0))
camera.parent = pivot
app.run()
```
